### ISSUE TITLE
Pressing setting tabs can easily trigger the notification drawer

### ISSUE THREAD

**Issue by OliverLew** [agateau/pixelwheels#140]
**Behavior**:

When the status bar is visible (see the screenshot below), it is very likely to touch the status bar and trigger the notification drawer.

![pixelwheel_mistouch](https://user-images.githubusercontent.com/12032219/136926597-b5b2d531-6c5e-43b8-bf2b-21403994d30d.png)

The status bar could be visible in a lot of cases:
- Right after unlocking screen
- Switched from another app
- During and after some popup (e.g., share log)

Note: this status bar is not when you swipe down at the top of the screen. In that case, the status bar has a semi-transparent background and hide in seconds. In my case this status bar has transparent background, and will persist if there is no interference.

**Possible solution**:

Initially, I wanted to request adding some padding above the tab buttons. That might still be a good thing to do, regardless of this  issue. However, as I found out, other full-screen games I have won't show the status bar in those occasions, so there are fewer chances to trigger the notification drawer.

So, is there something (or lack of) in pixelwheel that's causing the status bar to show? IMHO, make it not show up might be the proper way to fix the issue.

Edit: forgot to mention, I am using Lineage OS 18.1, which based on Android 11. The pixel wheels version is 0.21.0.

**Comment by agateau** [comment]
Good news is: I can reproduce it in Android emulator. I did it like this:
- Create a VD with an Android 11 ROM
- Start Pixel Wheels
- Go to "Settings > Under the hood > Report bug"
- Click on Back

Bad news: I don't know yet how to fix it. It looks a bit like [that bug][1] which was introduced in Android 11, but in our case the status bar is transparent, not fully opaque, so this might be a different bug.

[1]: https://support.google.com/pixelphone/thread/70424972/navigation-status-bar-not-disappearing-in-full-screen-apps-and-games-in-android-11-final-release?hl=en

**Comment by OliverLew** [comment]
I tried the games available on my phone again, and found there is another game with similar behavior: [mindustry](https://github.com/Anuken/Mindustry/).

If I am not wrong, one common thing between the two games is they are made with libgdx ([itch.io](https://itch.io/games/made-with-libgdx) also tells me so). I searched issues or PRs in [libgdx repo](https://github.com/libgdx/libgdx), and indeed found some related to Android status bar. But I am not good enough to confirm those are even slightly related, so I won't link them here. Maybe this issue is from libgdx instead of within this app.

For now I am using a trick to temporarily hide the status bar, which is to swipe from side or bottom of the screen to trigger the status bar again, after this the status bar will hide.
